docs/operator-manual/argocd-cmd-params-cm.yaml

```diff
@@ -179,6 +179,10 @@ data:
   applicationsetcontroller.namespaces: "argocd,argocd-appsets-*"
   # Path of the self-signed TLS certificate for SCM/PR Gitlab Generator
   applicationsetcontroller.scm.root.ca.path: ""
+  # Enforce security by restricting the list of allowed SCM providers (default "" is all SCM providers).
+  # Setting this field is required when using ApplicationSets-in-any-namespace, to prevent users from
+  # sending secrets from `tokenRef`s to disallowed `api` domains.
```

Comment:
nit: Add a statement to mention that this needs to be a comma-separated list of strings?